optuna_dashboard/_app.py

```diff
@@ -102,8 +102,8 @@ def api_meta() -> dict[str, Any]:
     @app.get("/api/studies")
     @json_api_view
     def list_study_summaries() -> dict[str, Any]:
-        summaries = get_study_summaries(storage)
-        serialized = [serialize_study_summary(summary) for summary in summaries]
+        studies = get_studies(storage)
+        serialized = [serialize_frozen_study(s) for s in studies]
         return {
             "study_summaries": serialized,
```

Comment:
Should I change the key name of `study_summaries` and `study_summary` to something?

Reply:
Could you add a todo comment? For now, we cannot introduce the breaking changes in JSON Web APIs since our Jupyter Lab extension uses it.